plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,30 @@ struct MutantTimeoutCtx {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("mutationStatusId")
+struct MarkedMutant {
+    @ColumnName("st_id")
+    ulong mutationStatusId;
+
+    uint line;
+
+    uint column;
+
+    string path;
+
+    @ColumnName("status")
+    ulong to_status;
+
+    @ColumnName("time")
+    SysTime time;
+
+    @ColumnName("rationale")
```

Comment:
You have remove this UDA. the name should be, automatically, the same as the field name